### PR TITLE
Version Bump to 0.2.11

### PR DESCRIPTION
**Pull request by elbrujohalcon**

### CODE REVIEW

**Comment by elbrujohalcon** [comment]
I actually piggy-backed a couple of dep adjustments in this PR.
⚠️  Don't do that at home, kids. ⚠️

File: rebar.config [hunk 1]
```diff
@@ -21,17 +20,15 @@
 {deps_dir, "deps"}.
 {deps,
  [
-  {lager,      ".*",  {git, "https://github.com/basho/lager.git",          "3.0.2"}},
   {elvis_core, ".*",  {git, "https://github.com/inaka/elvis_core",         "0.2.7"}},
```
Comment (harenson) [inline rebar.config]: You updated `elvis_core` in the `Makefile`, so please update it here. Also add `katana_code` here, please.

File: rebar.config [hunk 2]
```diff
@@ -21,17 +20,15 @@
...
   {getopt,     "0.*", {git, "https://github.com/jcomellas/getopt.git",     "v0.8.2"}},
   {meck,       "0.*", {git, "https://github.com/eproxus/meck.git",         "0.8.2"}},
   {jiffy,      "0.*", {git, "https://github.com/davisp/jiffy.git",         "0.14.3"}},
-  {ibrowse,    "4.*", {git, "https://github.com/cmullaparthi/ibrowse.git", "v4.1.2"}},
   {aleppo,     "0.*", {git, "https://github.com/inaka/aleppo.git",         "0.9.0"}},
   {zipper,     ".*",  {git, "https://github.com/inaka/zipper.git",         "0.1.2"}},
```
Comment (harenson) [inline rebar.config]: Remove `zipper`.

File: rebar.config [hunk 3]
```diff
@@ -21,17 +20,17 @@
...
+  {ibrowse,     "4.*", {git, "https://github.com/cmullaparthi/ibrowse.git", "v4.1.2"}},
+  {aleppo,      "0.*", {git, "https://github.com/inaka/aleppo.git",         "0.9.10"}},
+  {egithub,     ".*",  {git, "https://github.com/inaka/erlang-github.git",  "0.2.2"}},
+  {katana,      ".*",  {git, "https://github.com/inaka/erlang-katana.git",  "0.2.23"}},
+  {katana_code, ".*",  {git, "https://github.com/inaka/katana-code.git",    "0.0.3"}}
  ]
 }.
 {escript_name, "elvis"}.
-{escript_incl_apps, [getopt, jiffy, ibrowse, aleppo, zipper, katana]}.
+{escript_incl_apps, [getopt, jiffy, aleppo, zipper, katana, katana_code]}.
```
Comment (harenson) [inline rebar.config]: Please, remove `zipper` from this list, it is not an `elvis` dependency.

**Pull request merged**